ingest/workflow/snakemake_rules/nextclade.smk

```diff
@@ -61,6 +61,7 @@ rule join_metadata_clades:
         metadata="data/metadata.tsv",
     params:
         id_field=config["transform"]["id_field"],
+        nextclade_id_field=config["transform"]["nextclade_id_field"],
```

Comment:
_non-blocking_

It's slightly weird to me that this param is under `transform` since it is not being used in the transform rule. I think it's reasonable to move this to a top level config key, but that's just me being nit-picky.

Reply:
I think that's a reasonable and quick change. Perhaps even defining a nextclade config section: https://github.com/nextstrain/monkeypox/pull/207/commits/9817e6e51f3d691a27270b155dec7dfd44623738

Reply:
That's the direction I'm pushing in the [pathogen-repo-template](https://github.com/nextstrain/pathogen-repo-template/pull/7) as well! 

I think it still works well here even though we are technically using [two different Nextclade datasets](https://github.com/nextstrain/monkeypox/blob/3e836176c959803263dd260726d3325e1bf5d0c8/ingest/workflow/snakemake_rules/nextclade.smk#L2-L17).